IndexedDB/idbobjectstore_createIndex15-autoincrement.htm

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>IDBObjectStore.createIndex() - AutoIncrement in Compound Index</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="support.js"></script>
+<script>
+  indexeddb_test(
+    function(t, db, txn) {
+      // Store 1 - No auto-increment
+      var store1 = db.createObjectStore("Store1", {keyPath: "id"});
```

Comment:
Since the `store` variables are not shared across tests, maybe just call it `store` in both?